include/http/router.js

```diff
@@ -145,23 +146,39 @@ module.exports = function (pb) {
                     .use(this.router.allowedMethods());
 
                 this._addDefaultMiddleware();
-
-                this.__server = this.app.listen(port, () => {
-                    pb.log.info('PencilBlue is ready!');
-                });
+                const config = {
```

Comment:
is this just a test configuration object ? because your are binding to a hardcoded port right now

Reply:
Yes, right now is hardcoded. We need a card to enable https with KOA.

Reply:
Just pushed a new version with the card [TN-813](https://careerbuilder.atlassian.net/browse/TN-813) that has the protocol server configurable like in the old implementation.